//package com.sincetimes.redwar.game.comnunicate.protocol;


message RseQueryStarsBookmarks
{
	repeated Bookmarks Bookmarks = 1;
}

message Bookmarks
{
	optional int32 ret = 1;
}